changelog.txt

```diff
@@ -1,3 +1,14 @@
+1.1.1-preview
+=============
+This release marks the third preview of the library which brings in several features and changes -

```

Comment:
this is not the third preview. this is a patch release. just say, "This release contrains"

Reply:
done.